### ISSUE TITLE
Training on Custom Objects in Linux using python

### ISSUE THREAD

**Issue by prateethvnayak** [AlexeyAB/darknet#1290]
All the steps under the training the yolov3 for custom objects pertains to making obj.names and obj.data and all the requirements in the build/darknet/x64/ 
However, at the end it requires to use darknet.cmd ! Is there a way for training custom objects in python ? The instructions dont seem to exist or is it possible !?

**Comment by dexception** [comment]
You need to make you own obj.names and obj.data files to train.

**Comment by prateethvnayak** [comment]
Yes, I did that. I was talking about the path where it should be saved. Whether in the build folder as I have stated above ? or in the darknet/data folder ? Since the py-executable is not available in the build folder but is in darknet/
Thanks
